### DISCUSSION TITLE
Zoom meetings για διευκρινίσεις και απορίες

### DISCUSSION THREAD

**Discussion by mariosconsta** [courses-ionio/hci#1599] [Announcements]
Θα μπορούσε κάποιος καθηγητής ή υπεύθυνος του μαθήματος να κάνουμε ένα μικρό zoom meeting, έστω για 10-15 λεπτά διότι έχω απορίες για το μάθημα και δεν βρίσκομε στο νησί για να σας ρωτήσω από κοντά; Ξέρω μπορώ να γράψω εδώ ότι απορία έχω αλλά θα προτιμούσα zoom ή κάποια άλλη πλατφόρμα να μιλήσουμε κανονικά. Είναι το μόνο μάθημα που μου μένει για να δηλώσω πτυχιακή και για αυτό θα ήθελα να τα πάω όσο καλύτερα μπορώ εξ αποστάσεως.
Ευχαριστώ!

**Comment by epidrome** [comment]
με στόχο να εξυπηρετήσουμε όσο καλύτερα μπορούμε όσους περισσότερους γίνεται,

ούτε στα μαθήματα με φυσική παρουσία κάνουμε απορίες! 

για αυτό έχουμε τα ντισκασιονς εδώ

μια εύστοχη ερώτηση, με καλό τίτλο-περιγραφή, αργά ή γρήγορα, θα βρει μια απάντηση εδώ δημόσια για το καλό όλων!
